source/libraries/CompileSolidity.ts

```diff
@@ -72,26 +84,26 @@ class SolidityContractCompiler {
     }
 
     private async generateCompilerInput(): Promise<CompilerInput> {
+        const filePaths: any = await recursiveReadDir(this.contractInputDirectoryPath, [this.ignoreFile]);
```

Comment:
You can delete this `: any`.  `recursiveReadDir()` correctly returns `Promise<string[]>` and when I tested everything works correctly with that.